command_line/image_viewer.py

```diff
@@ -44,6 +44,8 @@
   .type = bool
 show_all_pix = True
   .type = bool
+show_thresh_pix = False
```

Comment:
Explicit parameter name better than saving 3 characters:
```suggestion
show_threshold_pix = False
```